lib/tasks/ci.rake

```diff
@@ -2,7 +2,11 @@ ZIP_FILE = 'https://github.com/projecthydra/hydra-jetty/archive/v7.3.1.zip'
 
 desc "Run Continuous Integration Suite (tests, coverage, docs)"
 task :ci => ['jetty:clean', 'jetty:config'] do
-  Rails.env = "test"
+  unless Rails.env.test?
+    system('RAILS_ENV=test rake ci')
```

Comment:
I believe setting `Rails.env` is insufficient for getting our environment correctly set up. We do this same pattern in other places as well (see Searchworks, for one).

Yeah, they'll run twice, but it isn't that expensive.